post_processing/ci_plots.jl

```diff
@@ -1311,3 +1311,50 @@ function make_plots(::EDMFSpherePlots, output_paths::Vector{<:AbstractString})
         MAX_NUM_ROWS = 4,
     )
 end
+
+
+function make_plots(
+    ::Val{:gcm_driven_scm},
+    output_paths::Vector{<:AbstractString},
+)
+    simdirs = SimDir.(output_paths)
+    short_names_2D = [
+        "rlut",
+        "rlutcs",
+        "rsut",
+        "rsutcs",
+        "clwvi",
+        "lwp",
+        "clivi",
+        "dsevi",
+        "clvi",
+        "prw",
+        "hurvi",
+    ]
+    short_names_3D = ["husv", "thetaa", "ta", "hur", "hus", "clw", "cl"]
+    reduction = "inst"
+    vars_2D = map_comparison(simdirs, short_names_2D) do simdir, short_name
+        average_xy(get(simdir; short_name, reduction))
+    end
+    vars_3D = map_comparison(simdirs, short_names_3D) do simdir, short_name
+        data = window(
+            get(simdir; short_name, reduction),
+            "z",
+            left = 0,
+            right = 4000,
+        )
+        return average_xy(data)
+    end
```

Comment:
If your simulation is just a column, you can `slice(var, x = 0, y = 0)` to get its value